bioptim/limits/penalty.py

```diff
@@ -614,6 +645,27 @@ def minimize_all_controls(penalty: PenaltyOption, pn: PenaltyNodes):
                 penalty.sliced_target = target[:, i] if target is not None else None
                 penalty.type.get_type().add_to_penalty(pn.ocp, pn, val, penalty)
 
+        @staticmethod
+        def minimize_control_derivatives(penalty: PenaltyOption, pn: PenaltyNodes):
```

Comment:
I don't think we usually put the names in plural (`minimize_control_derivatives`). If so, ignore this comment